### PR TITLE
Fix bar being freezed after moving left or right

### LINKED ISSUE #17: Move left or right in menu bar may cause the bar freezed

### PR DESCRIPTION
**Pull request by micielski**
This seems to fix #17

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -146,7 +146,9 @@ impl<T: Clone> MenuState<T> {
     ///
     /// down highlights "sub item 2"
     pub fn down(&mut self) {
-        if self.active_depth() == 1 {
+        if self.active_depth() == 0 {
```
Comment (shuoli84) [inline src/lib.rs]: should change the code in right?

Comment (shuoli84) [reply src/lib.rs]: oh right, "right" is going deep, so it won't pop the root_item

Comment (shuoli84) [reply src/lib.rs]: the change made in "left" should fixed the problem. this change is not required?

Comment (micielski) [reply src/lib.rs]: Oh, I meant to do this in right(), not down().....

**Review by shuoli84 — APPROVED** [review]
LGTM

**Comment by shuoli84** [comment]
Thanks!

**Pull request merged**